ads/medianet.js

```diff
@@ -132,7 +132,7 @@ function loadHBTag(global, data, publisherUrl, referrerUrl) {
     let currentParam = '';
     for (let i = 0; i < allParams.length; i++) {
       currentParam = allParams[i];
-      if (dfpParams.indexOf(currentParam) === -1 && data[currentParam]) {
+      if (!dfpParams.includes(currentParam) && data[currentParam]) {
```

Comment:
So everything in this file cannot be updated, because the ads directory is run in foreign iframes. This means they will not have the Array#includes polyfill installed in them.